cv19ResSupportV3/V3/Gateways/HelpRequestGateway.cs

```diff
@@ -326,7 +326,7 @@ public List<HelpRequestEntity> GetCallbacks(CallbackRequestParams requestParams)
                     .ToList();
                 if (!string.IsNullOrWhiteSpace(requestParams.Master))
                 {
-                    return response.Where(x => x.RecordStatus != null && x.RecordStatus.ToUpper() == "MASTER").ToList();
+                    return response.Where(x => x.RecordStatus != null && x.RecordStatus.Replace(" ", "").ToUpper() == "MASTER").ToList();
```

Comment:
Would it make sense to use Trim() instead?

Reply:
I tried to factor in any accidential whitespace including "M ASTER" for example.  You can never tell what can happen while we are updating manually.